Hyperdrive/HyperBlueprint.swift

```diff
@@ -127,13 +88,13 @@ public class HyperBlueprint : Hyperdrive {
   // MARK: Entering an API
 
   /// Enter an API from a blueprint hosted on Apiary using the given domain
-  public class func enter(# apiary: String, baseURL:NSURL? = nil, completion: (HyperBlueprintResult -> Void)) {
+  public class func enter(apiary  apiary: String, baseURL:NSURL? = nil, completion: (HyperBlueprintResult -> Void)) {
```

Comment:
Just want to point out, this means that the parameter `apiary` is named `apiary`.

Which results in using the API like:

```swift
HyperBlueprint.enter(apiary: "pollsapp")
```

Instead of:

```swift
HyperBlueprint.enter("pollsapp")
```

In Swift 1.0, `#` use to be equivalent.